modules/common/Utils.py

```diff
@@ -42,15 +44,24 @@ def gsutil_multi_copy_to(self, destination_bucket):
         # cmd_result = shutil.which("gsutil")
         # cmd = "gsutil -q -m cp -r " + self.yaml.output_dir + "/* gs://" + destination_bucket + "/"
         # subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
-        logger.debug("gsutil_multi_copy_to - using gsutil for source '{}', destination '{}'"
-                     .format(os.path.join(self.output_dir, "*"), "gs://{}/".format(destination_bucket)))
-        proc = subprocess.Popen(["gsutil",
-                                 "-m",
-                                 "cp",
-                                 "-r",
-                                 os.path.join(self.output_dir, "*"),
-                                 "gs://{}/".format(destination_bucket)],
-                                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        logger.debug(
+            "gsutil_multi_copy_to - using gsutil for source '{}', destination '{}'".format(
+                os.path.join(self.output_dir, "*"),
+                "gs://{}/".format(destination_bucket),
+            )
+        )
+        proc = subprocess.Popen(
```

Comment:
Is there anyway to disable your Python formatter until we have a defined formatting strategy for our Python code base?